spec/registrant_spec.rb

```diff
@@ -0,0 +1,40 @@
+require './lib/registrant'
+
+RSpec.describe Registrant do 
+  describe "#initialize" do 
+    it "exists" do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      registrant_2 = Registrant.new('Penny', 15 )
+      
+      expect(registrant_1).to be_a Registrant
+      expect(registrant_2).to be_a Registrant
+    end
+    
+    it "has an age, name and permit" do 
```

Comment:
Since registrants do not have a permit by default, you could consider calling it "permit status" or having a separate test for this attribute called `it "might have a permit" do`